components/kyma-environment-broker/internal/process/input/input.go

```diff
@@ -40,13 +40,17 @@ type Config struct {
 }
 
 type RuntimeInput struct {
+	muOptionalComponents sync.Mutex
+	muLabels             sync.Mutex
+	muOverrides          sync.Mutex
+
 	provisionRuntimeInput gqlschema.ProvisionRuntimeInput
 	upgradeRuntimeInput   gqlschema.UpgradeRuntimeInput
 	upgradeShootInput     gqlschema.UpgradeShootInput
-	mutex                 *nsync.NamedMutex
-	overrides             map[string][]*gqlschema.ConfigEntryInput
-	labels                map[string]string
-	globalOverrides       []*gqlschema.ConfigEntryInput
+	//mutex                 *nsync.NamedMutex
```

Comment:
nit: remove this line instead of keeping as a comment?

Reply:
yes, the PR was not ready for review, I forgot to make it draft